lib/pacemaker/pcmk_sched_allocate.c

```diff
@@ -280,11 +280,19 @@ check_action_definition(pe_resource_t * rsc, pe_node_t * active_node, xmlNode *
        && digest_secure
        && digest_data->digest_secure_calc
        && strcmp(digest_data->digest_secure_calc, digest_secure) == 0) {
-        if (pcmk_is_set(data_set->flags, pe_flag_stdout)) {
-            printf("Only 'private' parameters to " PCMK__OP_FMT
-                   " on %s changed: %s\n",
-                   rsc->id, task, interval_ms, active_node->details->uname,
-                   crm_element_value(xml_op, XML_ATTR_TRANSITION_MAGIC));
+        if (pcmk_is_set(data_set->flags, pe_flag_print)) {
+            if (data_set->priv != NULL) {
+                pcmk__output_t *out = data_set->priv;
+                out->info(out, "Only 'private' parameters to "
+                          PCMK__OP_FMT " on %s changed: %s", rsc->id, task,
+                          interval_ms, active_node->details->uname,
+                          crm_element_value(xml_op, XML_ATTR_TRANSITION_MAGIC));
+            } else {
+                printf("Only 'private' parameters to " PCMK__OP_FMT
+                       " on %s changed: %s\n",
+                       rsc->id, task, interval_ms, active_node->details->uname,
+                       crm_element_value(xml_op, XML_ATTR_TRANSITION_MAGIC));
+            }
```

Comment:
This is all internal API so I don't think we have to care about when data_set->priv is NULL